.gitignore

```
@@ -14,3 +14,18 @@ api-gen/build/
 
 # IDEs
 .vscode
+# Generated files by hugo
+/public/
+/resources/_gen/
+/assets/jsconfig.json
+hugo_stats.json
+
+# Executable may be added to repository
+hugo.exe
+hugo.darwin
+hugo.linux
+
+# Temporary lock file while building
+/.hugo_build.lock
+/themes/**
+.DS_Store**
```

I think this is broken - simply referencing `.DS_Store` in the file is sufficient (any file with this name will match).

```suggestion
.DS_Store
```